tests/redpanda.nix

```diff
@@ -145,24 +145,31 @@ rebuildableTest {
     };
   };
 
+  # "reserve" 15GiB early by allocating it here. The key is to do it in small
+  # increments so nixbuild.net detects the lack of memory. Big increments go
+  # unnoticed by the OOM logic. This is exactly what happens with prodserver
+  # below. It requests too much memory at once, and nixbuild.net doesn't
+  # restart the build as expected.
   testScript = ''
+    x = []
+    for _ in range(1024**2):        # 1 MiB
+      x.append(bytearray(1024*15))  #       * 15 kiB = 15 GiB
+    x = []
```

Comment:
@Radvendii You will like this one :-)

Reply:
hahaha oh no

Reply:
Just waiting for the python update that optimizes this away and the bug comes back